package.json

```diff
@@ -38,13 +39,15 @@
     "nprogress": "^1.0.0-1",
     "preact": "^10.6.6",
     "preact-compat": "^3.19.0",
+    "svg-pan-zoom": "^3.6.1",
     "vue": "^2.6.14",
     "vue-i18n": "^8.27.0",
     "vue-meta": "^2.4.0",
     "vue-router": "^3.5.3",
     "vue-the-mask": "^0.11.1",
     "vuetify": "^2.6.10",
-    "vuex": "^3.6.2"
+    "vuex": "^3.6.2",
+    "wasm-loader": "^1.3.0"
```

Comment:
Don't webpack loaders normally only need to be dev dependencies?

Reply:
Makes sense, will move.